tests/scripts/test-cluster-updates.sh

```
@@ -0,0 +1,65 @@
+#   5. Collect artefacts and clean up
+
+set -eou pipefail
+
+make install-deps && make install-deps-test
+
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+. "$DIR/utils.sh"
+
+# Install operator-sdk
+export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)